server.js

```diff
@@ -22,6 +23,12 @@ app.use(express.json());
 app.use('/test-jwt', testJWTRouter);
 app.use('/users', usersRouter);
 app.use('/profiles', profilesRouter);
+app.use('/pinstas', pinstaRouter)
```

Comment:
Should add `app.use(verifyToken);` before this line to make it protected, you also need to import verifyToken in this file `const verifyToken = require('./middleware/verify-token');
`